packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-table-manager-lambda/cfn-response.ts

```diff
@@ -0,0 +1,112 @@
+// The contents of this file were taken from the AWS CDK provider framework.
+// https://github.com/aws/aws-cdk/blob/c52ff08cfd1515d35feb93bcba34a3231a94985c/packages/aws-cdk-lib/custom-resources/lib/provider-framework/runtime/cfn-response.ts
```

Comment:
This is a 1:1 copy of the CDK provider framework `cfn-response.ts` file. No modifications were made.